jsAgent/tour.js

```
@@ -1,8 +1,328 @@
+     * Loads floating-ui libraries asynchronously.
+     * @param {function(object): void} cb - Callback function, called with the tour options object.
+     * @param {function(): void} errCb - Callback function, called on error.
+     * @returns {void}
+     */
+    loadFloatingUILibraries: function (cb) {
+        bw.util.loadScriptAsync(FLOATING_UI_CORE_URL, () => {
+            console.log("@floating-ui/core loaded successfully");
+            bw.util.loadScriptAsync(FLOATING_UI_DOM_URL, () => {
+                console.log("@floating-ui/dom loaded successfully");
+                const options = window.bwonboarddata.tour[0];
+                cb && cb(options);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)